src/SecurityManager.js

```diff
@@ -262,6 +326,18 @@ export async function accessSecretStorage(func = async () => { }, forceReset = f
             await cli.bootstrapSecretStorage({
                 getKeyBackupPassphrase: promptForBackupPassphrase,
             });
+
+            const keyId = Object.keys(secretStorageKeys)[0];
+            if (keyId && SettingsStore.getValue("feature_dehydration")) {
+                const dehydrationKeyInfo =
+                      secretStorageKeyInfo[keyId] && secretStorageKeyInfo[keyId].passphrase
+                      ? {passphrase: secretStorageKeyInfo[keyId].passphrase}
+                      : {};
+                console.log("Setting dehydration key");
+                await cli.setDehydrationKey(secretStorageKeys[keyId], dehydrationKeyInfo, "Backup device");
```

Comment:
Is it correct to call `setDehydrationKey` perhaps several times a session? `accessSecretStorage` is meant to be safe to call whenever anything 4S related is needed, so it could be any number of times.

Reply:
It'll be fine because `setDehydrationKey` checks to see if the call changes the key, and if it uses the same key, it will do nothing.